mycli/sqlexecute.py

```diff
@@ -61,13 +64,19 @@ def connect(self, database=None, user=None, password=None, host=None,
             '\tlocal_infile: %r'
             '\tssl: %r',
             database, user, host, port, socket, charset, local_infile, ssl)
+        conv = {
+                FIELD_TYPE.TIMESTAMP: lambda obj: (convert_mysql_timestamp(obj) or '0000-00-00 00:00:00'),
```

Comment:
Depending on the MySQL server version (and how old the data is), there could be invalid dates/times such as `2007-02-31` which will return `None` from PyMySQL's converters. Instead of hard-coding the fallback value as `0000-00-00...`, how about we just return `obj` so that the original value from the database is shown?